AFQ/tests/test_api.py

```diff
@@ -168,6 +168,25 @@ def test_AFQ_data():
         myafq.export_rois()
 
 
+def test_AFQ_anisotropic():
+    """
+    Test if API can run without prealign
+    """
+    tmpdir = nbtmp.InTemporaryDirectory()
+    afd.organize_stanford_data(path=tmpdir.name)
+    bids_path = op.join(tmpdir.name, 'stanford_hardi')
+    myafq = api.AFQ(
+        bids_path=bids_path,
+        dmriprep='vistasoft',
+        segmentation='freesurfer',
+        min_bval=1990,
+        max_bval=2010,
+        b0_threshold=50,
+        reg_template="mni_T1",
+        reg_subject="power_map")
+    myafq.export_rois()
```

Comment:
Can we do something more specific here to check that the selection works as expected? I guess we can call individual (private) methods of this object once it’s initialized.

Reply:
Is this similar to what you had in mind?

Reply:
Can we maybe check that the APM is where we'd expect it to be at this point?

Reply:
I am now checking that the APM map is roughly similar (rounded to the ten's place) to the map in file_dict. Is there a better way to check the mapping?